politeiad/cmd/politeia_verify/README.md

```diff
@@ -1,42 +0,0 @@
-# politeia_verify
```

Comment:
I am not sure I agree with killing the one off verify tool in favor of doing it elsewhere. This tool exists for the very specific reason that one can save all data and verify independently of infrastructure. I am ok deleting the code out of this tool and importing it from elsewhere but the frontend tool should remain in existence. People currently rely on this tool and I don't feel that we should force them to stop using it.

Reply:
> I am ok deleting the code out of this tool and importing it from elsewhere but the frontend tool should remain in existence.

That's what I did.  I wanted a verification tool that can be used to verify the bundles downloaded from the gui (proposal bundle, comments bundle, proposal timestamps bundle, comment timestamps bundle).  To avoid the confusion of having multiple verification tools, I moved this functionality into the `politeia` tool, the `politeia verify` command now does the same thing that this tool used to do, and then created a verify tool in `politeiawww/cmd` that is used to verify data returned from the politeiawww API and downloaded from the gui.